src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/BundleViewPathPass.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\TwigBundle\DependencyInjection\Compiler;
+
+use Symfony\Component\Config\Resource\FileExistenceResource;
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+/**
+ * Registers the bundles view paths.
+ */
+final class BundleViewPathPass implements CompilerPassInterface
+{
+    public function process(ContainerBuilder $container)
+    {
+        $twigFilesystemLoaderId = 'twig.loader.filesystem';
+        if ($container->hasDefinition($twigFilesystemLoaderId)) {
+            $twigFilesystemLoaderDefinition = $container->getDefinition($twigFilesystemLoaderId);
+        } elseif ($container->hasAlias($twigFilesystemLoaderId)) {
+            $aliasedService = (string) $container->getAlias($twigFilesystemLoaderId);
+            if (false === $container->hasDefinition($aliasedService)) {
+                return;
+            }
+            $twigFilesystemLoaderDefinition = $container->getDefinition($aliasedService);
+        } else {
+            return;
+        }
```

Comment:
Could we use `$container->findDefinition()` to simplify it a bit? Also, there is no need for `if else return` IMO, I assume that `twig.loader.filesystem` alias **or** definition exist by default.

Reply:
Fixed